### ISSUE TITLE
Change the theme image to a Tyxan-related one.

### ISSUE THREAD

**Issue by tyxansimon** [tyxan/Tyxan-Theme#3]
Once we have a solid interface for the default front-end of the Tyxan theme, I'll then generate a new thumbnail.

However, maybe we just want this to be the Tyxan logo?
Though we want this theme to be white-label, right? Worth a discussion :)

**Comment by willevanstyxan** [comment]
Yeah, it needs to be 'white labelable' so defaults to a 'Tyxan' logo/image but the ability to both change the name of the theme and the image.

**Comment by tyxansimon** [comment]
When you say "change the name", you mean the front-facing stuff on the Appearance > Themes page, or the theme directory as well?

**Comment by willevanstyxan** [comment]
Preferably both but I doubt thats doable so maybe just the front facing stuff.

**Comment by tyxansimon** [comment]
Yeah the theme directory would be a manual process.
I'll make the image and theme name in Appearance > Themes manageable in the next [/assets/acf-fields/group_theme_options.json](theme options) commit.

**Comment by willevanstyxan** [comment]
Cool

**Comment by tyxansimon** [comment]
Re-opening as presumably best to close once fixed? Worth a test in the new workflow. Stay tuned 🤓
